src/main/java/seedu/address/commons/exceptions/IllegalValueException.java

```diff
@@ -13,7 +14,7 @@ public IllegalValueException(String message) {
 
     /**
      * @param message should contain relevant information on the failed constraint(s)
-     * @param cause of the main exception
+     * @param cause   of the main exception
```

Comment:
Looks like atypo

Reply:
It seems that checkstyle will auto-format like that